nixos/tests/prometheus-exporters.nix

```diff
@@ -241,7 +242,9 @@ let
         wait_for_open_port(80)
         wait_for_unit("prometheus-json-exporter.service")
         wait_for_open_port(7979)
-        succeed("curl -sSf localhost:7979/metrics | grep -q 'json_test_metric 1'")
+        succeed(
+            "curl -sSf 'localhost:7979/metrics?target=http://localhost' | grep -q 'json_test_metric 1'"
```

Comment:
This test doesn't seem to pass: https://github.com/NixOS/nixpkgs/pull/108179/checks?check_run_id=1634971193

```
Exception: command `curl -sSf 'localhost:7979/metrics?target=http://localhost' | grep -q 'json_test_metric 1'` failed (exit code 1)
```

Reply:
Oh yeah, I missed switching to the probe endpoint. Thanks for the hint :+1: